mantidimaging/gui/windows/live_viewer/view.py

```diff
@@ -0,0 +1,78 @@
+# Copyright (C) 2023 ISIS Rutherford Appleton Laboratory UKRI
+# SPDX - License - Identifier: GPL-3.0-or-later
+from __future__ import annotations
+from pathlib import Path
+from typing import TYPE_CHECKING, Optional
+
+from PyQt5.QtGui import QPixmap
+from PyQt5.QtWidgets import QPushButton, QVBoxLayout, QCheckBox, QLabel
+from tifffile import tifffile
+
+from mantidimaging.core.utility import finder
+from mantidimaging.gui.mvp_base import BaseMainWindowView
+from mantidimaging.gui.widgets.dataset_selector import DatasetSelectorWidgetView
+from mantidimaging.gui.windows.image_load_dialog import ImageLoadDialog
+
+from mantidimaging.gui.widgets import RemovableRowTableView
+from .live_view_widget import LiveViewWidget
+
+from .presenter import LiveViewerWindowPresenter
+
+import numpy as np
+
+if TYPE_CHECKING:
+    from mantidimaging.gui.windows.main import MainWindowView  # noqa:F401  # pragma: no cover
+    from uuid import UUID
+
+
+class LiveViewerWindowView(BaseMainWindowView):
+    sampleStackSelector: DatasetSelectorWidgetView
+    normaliseStackSelector: DatasetSelectorWidgetView
+    image_load_dialog: Optional[ImageLoadDialog] = None
+
+    tableView: RemovableRowTableView
+
+    normaliseCheckBox: QCheckBox
+    imageLayout: QVBoxLayout
+    exportButton: QPushButton
+    normaliseErrorIcon: QLabel
+    _current_dataset_id: Optional['UUID']
+    normalise_error_issue: str = ""
+
+    def __init__(self, main_window: 'MainWindowView', live_dir_path: Path) -> None:
+        super().__init__(main_window, 'gui/ui/live_viewer_window.ui')
+        self.setWindowTitle("Mantid Imaging - Live Viewer")
+        self.main_window = main_window
+        self._path = live_dir_path
+
+        self.presenter = LiveViewerWindowPresenter(self, main_window)
+
+        self.main_window = main_window
+
+        icon_path = finder.ROOT_PATH + "/gui/ui/images/exclamation-triangle-red.png"
+        self.normalise_error_icon_pixmap = QPixmap(icon_path)
+
+        self.live_viewer = LiveViewWidget()
+
+        self.imageLayout.addWidget(self.live_viewer)
+        self.show_first_image()
+
+    @property
+    def path(self):
+        """Path to the directory containing the images"""
+        return self._path
+
+    @path.setter
+    def path(self, path: Path):
+        """Set the path to the directory containing the images to watch"""
+        self._path = path
```

Comment:
As we are not doing anything fancy with `path` it can just be a regular public attribute.